Property_map/include/CGAL/Dynamic_property_map.h

```diff
@@ -141,50 +190,52 @@ struct dynamic_property_map<G,halfedge_property_t<T> >
 
 
 template <typename G, typename T>
-struct dynamic_property_map<G,edge_property_t<T> >
+struct property_map<G, CGAL::dynamic_edge_property_t<T> >
 {
   typedef typename boost::graph_traits<G>::edge_descriptor edge_descriptor;
   typedef CGAL::internal::Dynamic_property_map<edge_descriptor,T> type;
   typedef type const_type;
 };
 
 template <typename G, typename T>
-struct dynamic_property_map<G,face_property_t<T> >
+struct property_map<G, CGAL::dynamic_face_property_t<T> >
 {
   typedef typename boost::graph_traits<G>::face_descriptor face_descriptor;
   typedef CGAL::internal::Dynamic_property_map<face_descriptor,T> type;
   typedef type const_type;
 };
 
+} // namespace boost
 
+namespace CGAL {
 
 template <typename T, typename G>
-typename dynamic_property_map<G,vertex_property_t<T> >::const_type
-add_property(vertex_property_t<T> prop, const G&)
+typename boost::property_map<G, dynamic_vertex_property_t<T> >::const_type
+get(CGAL::dynamic_vertex_property_t<T> prop, const G&)
```

Comment:
shall we take a const & for `prop`?

Reply:
@sloriot I don't know what to do here

Reply:
Maybe we can avoid a copy here by taking a const& to the map.

Reply:
and also for the other similar functions

Reply:
it has no storage so there is nothing to gain here.